src/views/catalog/templatescatalog/components/TemplatesCatalogDrawer/TemplatesCatalogDrawerCreateForm.tsx

```diff
@@ -64,6 +68,16 @@ export const TemplatesCatalogDrawerCreateForm: FC<TemplatesCatalogDrawerCreateFo
       startVM,
     } = useCreateDrawerForm(namespace, subscriptionData, authorizedSSHKey);
 
+    const [vmNameValidated, setVMNameValidated] = useState<ValidatedOptions>(
```

Comment:
Are you sure this should be a state ? Or we can just do a const that verify at each render?

Reply:
Good catch! Fixed!